modules/example-browser/package.json

```diff
@@ -34,12 +34,12 @@
     "karma-chrome-launcher": "^3.1.0",
     "karma-coverage-istanbul-reporter": "^3.0.3",
     "karma-mocha": "2.0.1",
-    "karma-webpack": "^4.0.2",
+    "karma-webpack": "^5.0.0",
     "ts-loader": "9.2.5",
     "ts-node": "^10.2.1",
     "tslib": "^2.2.0",
     "typescript": "^4.0.2",
-    "webpack": "^4.42.1",
+    "webpack": "^5.42.0",
```

Comment:
NPM 7 required this upgrade.

However, when webpack is compiled, it throws an error. 
Interestingly, I believe the compile error is a warning that `Array.from` is no longer injected into browsers that do not support it... I am not dead sure. I am still wrapping my head about what webpack does.

This does bring webpack in `example-browser` inline with the webpack in `integration-browser`.